rubber_duck.py

```diff
@@ -71,13 +78,21 @@ def __init__(self,
 
         self._metrics_handler = wrap_steps(metrics_handler)
 
+        self.discord_client = discord_client
+
+        self.workflow_manager = workflow_manager
```

Comment:
Is this only needed so `FeedbackWorkflow` can post an event? If so, I would pass in a function like:
```python
async def post_event(namespace, name, identity, method, value): ...
```
and then pass in `record_external_event` from the workflow manager (instead of passing in the whole workflow manager). 

Then pass this method to `FeedbackWorkflow`

Reply:
Should I use record_external_event function instead of send_event? Since I'm using workflow_manager not historian, don't I have to use send_event?